api/src/test/java/org/openmrs/test/BaseContextSensitiveTest.java

```diff
@@ -258,51 +258,64 @@ public void assumeOpenmrsModules(String module, String... modules) {
 		String[] allModules = ArrayUtils.addAll(modules, module);
 		assumeOpenmrsProfile(null, allModules);
 	}
-	
+
 	/**
 	 * Allows to ignore the test if the environment does not match the given OpenMRS version.
-	 * 
+	 *
 	 * @param openmrsPlatformVersion
 	 * @since 1.11.3, 1.10.2, 1.9.9
 	 */
 	public void assumeOpenmrsPlatformVersion(String openmrsPlatformVersion) {
 		assumeOpenmrsProfile(openmrsPlatformVersion);
 	}
-	
+
 	/**
 	 * Get the number of times this class has been loaded. This is a rough approx of how many tests
 	 * have been run so far. This can be used to determine if the test is being run in a standalone
 	 * context or if other tests have been run before.
-	 * 
+	 *
 	 * @return number of times this class has been loaded
 	 */
 	public Integer getLoadCount() {
 		return loadCount;
 	}
-	
+
 	/**
 	 * Used for runtime properties. The default is "openmrs" because most people will use that as
 	 * the default. If your webapp and runtime properties are under a different name, override this
 	 * method in your tests
-	 * 
+	 *
 	 * @return String webapp name to assume when looking up the runtime properties
 	 */
 	public String getWebappName() {
 		return "openmrs";
 	}
-	
+
+
+  /**
+   * Override this method to turn on/off the in-memory database. The default is to use the
+   * in-memory database. When this method returns false, the database defined by the runtime
+   * properties is used instead
+   *
+   * @return true/false whether or not to use an in memory database
+   */
+  public Boolean useInMemoryDatabase() {
+    return true;
+    return !"false".equals(System.getProperty("useInMemoryDatabase"));
+  }
+
```

Comment:
With your changes, this while will obviously not compile. I believe if your using an IDEA, that can be seen in its editor. Be sure to know the syntax of the Java SE.